packages/jira-adapter/src/adapter.ts

```diff
@@ -203,7 +207,7 @@ export default class JiraAdapter implements AdapterOperations {
 
     log.debug('going to run filters on %d fetched elements', elements.length)
     progressReporter.reportProgress({ message: 'Running filters for additional information' })
-    await this.createFiltersRunner().onFetch(elements)
+    const filterResult = (await this.createFiltersRunner().onFetch(elements) ?? {}) as FilterResult
```

Comment:
can we avoid the cast?
looks like this works
```suggestion
    const filterResult = await this.createFiltersRunner().onFetch(elements) || {}
```